src/lib/githubDataManager.ts

```
@@ -0,0 +1,357 @@
+/**
+ * GitHub Data Manager
+ *
+ * This module handles fetching, storing, and accessing Pokemon TCG data
+ * from the GitHub repository: https://github.com/PokemonTCG/pokemon-tcg-data
+ */
+
+import { PokemonCard, CardSet } from '@/lib/types';
+import { getWithExpiry, setWithExpiry, CACHE_TIMES, CACHE_KEYS, createCacheKey } from '@/lib/cacheUtils';
```

ESLint

[error] 'getWithExpiry' is defined but never used.

(@typescript-eslint/no-unused-vars)